src/index.html

```diff
@@ -9,10 +9,53 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="/src/style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
```

Comment:
Use semantic tags where possible. Instead of `div` with class `all__stars`, consider using a `section` or `article` tag to better define the content section.